pages/morse.js

```diff
@@ -178,12 +178,23 @@ export default function MorsePage() {
         onRightArrowClick()
       }
     }
+
+    const handlePaste = (e) => {
+      const text = e.clipboardData.getData('text')
+      memoOnMorseButtonClick(text)
```

Comment:
Let's discuss standalone redux action(s). Currently, other key presses do not have standalone redux actions.
I prefer two new actions:
- `onKeyPress`
  - then most of the logic inside `handleKeyDown` can be moved to reducer; that is what reducers are intended for
  - it can share logic with `onMorseButtonClick`
- `onPaste`
  - it can share logic with `onMorseButtonClick`

Also, cursor logic should be different than now. Cursor should be moved after the last inserted character (now it is moved after the first inserted character).